package.json

```diff
@@ -587,15 +587,19 @@
         ]
     },
     "dependencies": {
-        "@hubspot/local-dev-lib": "^1.13.1",
+        "@hubspot/local-dev-lib": "^3.1.0",
```

Comment:
I know I'm late to the party here so this may have already been discussed, but there are a lot of breaking changes between LDL v1 and v3 (for example, the return type of anything using `http` is going to be different) and from what I can tell that isn't accounted for in this PR.

It looks like you could maybe get away with not upgrading though?

Reply:
Agree on this comment -- upgrading here would cause problems elsewhere and would avoid upgrading if possible. However, I do see the corresponding PR with the LDD updates - so will start reading through over there.

Reply:
Merged in the PR to bump LDL.